eth/gasprice/gasprice_test.go

```diff
@@ -395,3 +395,13 @@ func TestSuggestTipCapMaxBlocksSecondsLookback(t *testing.T) {
 		expectedTip:     big.NewInt(92_212_529_424),
 	}, timeCrunchOracleConfig())
 }
+
+func TestSuggestTipCapIncludesExtraDataGas(t *testing.T) {
+	applyGasPriceTest(t, suggestTipCapTest{
+		chainConfig:     params.TestChainConfig,
+		numBlocks:       20,
+		extDataGasUsage: DefaultMinGasUsed,
+		genBlock:        testGenBlock(t, 100_000, 1),
+		expectedTip:     big.NewInt(83_603_561_239),
```

Comment:
Where does this number come from?

![image](https://github.com/user-attachments/assets/89d3d5d6-1eea-4ab0-b31e-0be7ec0bf35a)

Reply:
So, the tip is equal to: `blockGasCost` * `baseFee` / `totalGasUsed`... But I doubt anyone has actually calculated the expected values for these tests.

The number ends up coming from:
1. The base fee changing as blocks are produced
2. The block gas cost increasing as blocks are produced

(the `totalGasUsed` is constant throughout the test)

I added a couple comments saying what the test is actually doing.